test/unit/org/apache/cassandra/auth/AuthCacheTest.java

```diff
@@ -322,6 +322,41 @@ public void testCacheLoaderIsNotCalledOnGetAllWhenCacheIsNotEmpty()
         assertEquals(1, loadCounter);
     }
 
+    @Test
+    public void testMetricsOnCacheEnabled()
```

Comment:
Should we also check the numbers for the `entries' and `maxEntries' method calls? They do not seem to be covered by the tests.

Reply:
done